testing/test_pluginmanager.py

```diff
@@ -20,8 +19,19 @@ def pytestpm():
     return PytestPluginManager()
 
 
+@pytest.fixture
+def config_for_test():
+    from _pytest.config import get_config
+
+    config = get_config()
+    try:
```

Comment:
You don't need the `try/finally` btw, fixture teardown is always executed even if the `yield` block raises